driven/provider/adapter.go

```diff
@@ -378,6 +378,34 @@ func (a *Adapter) executeQuery(body io.Reader, pathAndParams string, method stri
 	return data, nil
 }
 
+//GetCalendarEvents gives the events of the user
+func (a *Adapter) GetCalendarEvents(userID string) ([]model.CalendarEvent, error) {
+	//params
+	queryParamsItems := map[string][]string{}
```

Comment:
Where do you put these params?

 - user_x
 - course_x
 - course_x
 - start and end date times
 - per_page - the events per page